wiki/osu!supporter/en.md

```diff
@@ -1,61 +1,64 @@
+- Ability to additionally skin osu!;
+- Custom [leaderboards](/wiki/Beatmap#leaderboards).
 
-In general, by buying osu!supporter tag, you will be directly supporting the game's development while obtaining some extra goodies. It must be noted that the extra features from osu!supporter will never affect gameplay, performance, or score to your advantage in anyway.
+The full list of perks and donation goals may be seen at the [osu!supporter promo page](https://osu.ppy.sh/home/support). It is important to note that the extra features from osu!supporter will never give advantage in gameplay, performance, or score.
 
-## Getting osu!supporter
+## Permanent benefits
 
-![osu!store product banner](img/store-product.jpg?1 "osu!supporter product banner from the osu!store")
+The permanent benefits of the osu!supporter are:
 
-To get osu!supporter, sign in (or [register](/wiki/Registration)), then go to the [store page](https://osu.ppy.sh/store/products/supporter-tag). Use the slider or the month buttons (underneath the slider) to select the duration of the osu!supporter tag. A different duration will change the price you will pay. The currency used is in United States dollars (USD). If your country uses a different currency, please use a currency converter such as [OANDA](https://www.oanda.com/currency/converter/) or [Google](https://www.google.com/search?q=usd+exchange+rate) to check the price. The prices displayed on the osu!store page are by per individual account.
+- Ability to change the username once for free;
+- The editable `me!` section of the user profile, previously called a userpage.
 
-### Selecting the duration
+## Badge
 
-![Buying osu!supporter](img/selecting-duration.jpg?1 "Selecting the user and duration for osu!supporter")
+![Profile with osu!supporter](img/userpage.png?1 "Profile with osu!supporter")
 
-The usercard displayed above the slider is the user you are buying osu!supporter for. By default, it should be your usercard. To gift to a friend, enter your friend's username (not user ID) into the grey box below the usercard. Once this is done, the usercard will change to match the username you had entered. From here, you can then select the duration of osu!supporter you want to gift, but keep in mind that there is no promotion for group gifting.
+![Usercard with osu!supporter](img/usercard.png?1 "User card with osu!supporter")
 
-You can only buy or gift osu!supporter to one user at a time within this screen. To gift another user, you will need to go back to the [osu!supporter store page](https://osu.ppy.sh/store/products/supporter-tag) and repeat the steps for adding another user.
+The badge can be found next to the user avatar and on the usercard next to the country flag. The amount of hearts in the profile depends on how long a user has supported the game in total<!-- source: https://github.com/ppy/osu-web/blob/ac227d6abbd48630ebbf428c4e3372f388004ee5/app/Models/User.php#L1733-L1750 -->:
```

Comment:
There are more places it can appear in, such as the forums

Note that the code suggests that it only counts valid tag purchases, meaning gifted supporter would only ever account for the first heart... ?

```suggestion
Supporters get a badge with hearts on it, usually found next to their username. The amount of hearts depends on the length of osu!supporter the user has acquired in total<!-- source: https://github.com/ppy/osu-web/blob/ac227d6abbd48630ebbf428c4e3372f388004ee5/app/Models/User.php#L1733-L1750 -->:
```